plugins/vst_base/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ IF(LMMS_SUPPORT_VST)
 INCLUDE(BuildPlugin)
 
 IF(LMMS_BUILD_WIN32)
-	ADD_DEFINITIONS(-DPTW32_STATIC_LIB)
+	ADD_DEFINITIONS(-DPTW32_STATIC_LIB -std=c++0x)
```

Comment:
GCC supports -std=c++14 ; so I guess -std=c++0x includes C++14 and some features of C++17.